Sources/PointFree/Episode/EpisodeModel.swift

```diff
@@ -1,5 +1,6 @@
 public struct Episode {
   var blurb: String
+  var codeSampleDirectory: String
```

Comment:
a directory inside https://github.com/pointfreeco/episode-code-samples corresponding to that episode's code sample